src/client.js

```diff
@@ -541,6 +543,61 @@ class Client {
          * @type {Campaign.Application}
          */
         this.application = null;
+
+        // temporary start in 2000
+        this.lastTime = "2000-01-01T00:00:00.000";
+        this.buildNumber = "0"; // temporary build number start
+        setInterval(() => {
+
+          console.log("refresh");
+          const that = this;
+          const soapCall = this._prepareSoapCall("xtk:session", "GetModifiedEntities", true, this._connectionParameters._options.extraHttpHeaders);
```

Comment:
THis is a new API right? THe SDK must work with all Campaign versions. If the build number/version does not have this API, then the auto refresh should simply be disabled